gym/wrappers/order_enforcing.py

```diff
@@ -1,18 +1,48 @@
+"""Wrapper to enforce the order of environments."""
 import gym
 
 
 class OrderEnforcing(gym.Wrapper):
-    """This will produce an error if `step` is called before an initial `reset`"""
+    """A wrapper that will produce an error if `step` is called before an initial `reset`.
+
+    Example:
+        >>> from gym.envs.classic_control import CartPoleEnv
+        >>> env = CartPoleEnv()
+        >>> env = OrderEnforcing(env)
+        >>> env.step(0)
+        ResetNeeded: Cannot call env.step() before calling env.reset()
+        >>> env.render()
+        ResetNeeded: Cannot call env.render() before calling env.reset()
+        >>> env.reset()
+        >>> env.render()
+        >>> env.step(0)
+    """
 
     def __init__(self, env):
+        """A wrapper that will produce an error if :meth:`step` is called before an initial :meth:`reset`."""
         super().__init__(env)
         self._has_reset = False
 
     def step(self, action):
-        assert self._has_reset, "Cannot call env.step() before calling reset()"
-        observation, reward, done, info = self.env.step(action)
-        return observation, reward, done, info
+        """Steps through the environment with :param:`kwargs`."""
+        assert self._has_reset, "Cannot call env.step() before calling env.reset()"
+        return self.env.step(action)
 
     def reset(self, **kwargs):
+        """Resets the environment with :param:`kwargs`."""
```

Comment:
:param: doesn't seem to work with sphinx

Reply:
Good spot